graph/telemetry/istio/appender/waypoint.go

```diff
@@ -0,0 +1,62 @@
+package appender
+
+import (
+	"context"
+	"strings"
+
+	"github.com/kiali/kiali/business"
+	"github.com/kiali/kiali/config"
+	"github.com/kiali/kiali/graph"
+	"github.com/kiali/kiali/log"
+)
+
+const WaypointAppenderName = "hideWaypoint"
+const WaypointSuffix = "-istio-waypoint"
+
+type WaypointAppender struct {
+	GraphType string
```

Comment:
```suggestion
	Waypoint bool
```

You can use this to decide whether to execute the logic.

Reply:
Changes done based on these comments